kimchi/src/circuits/polynomials/chacha.rs

```diff
@@ -539,21 +539,24 @@ mod tests {
         let d = D::new(1024).unwrap();
 
         let pt = F::rand(rng);
-        let mut eval = || ProofEvaluations {
-            w: array_init(|_| F::rand(rng)),
-            z: F::rand(rng),
-            s: array_init(|_| F::rand(rng)),
-            generic_selector: F::zero(),
-            poseidon_selector: F::zero(),
-            lookup: Some(LookupEvaluations {
-                sorted: (0..(lookup_info.max_per_row + 1))
-                    .map(|_| F::rand(rng))
-                    .collect(),
-                aggreg: F::rand(rng),
-                table: F::rand(rng),
-                runtime: None,
-            }),
+        let mut eval = || {
+            ProofEvaluations::new(
```

Comment:
there really ought to be a `ProofEvaluations::rand_for_test()` or smthg

Reply:
Perhaps it should allow you to optionally set the witness, selector and lookups for the test also